### ISSUE TITLE
Mac OS X Anaconda-based qmake missing mkspecs and by default looks in /Users/aaronmeurer

### ISSUE THREAD

**Issue by spearsem** [ContinuumIO/anaconda-issues#231]
```
ely_spears@macbook:~/workspace/learnsmart/flow/QtByteRunner$ ./build.sh 
Could not find mkspecs for your QMAKESPEC(macx-g++) after trying:
	/Users/aaronmeurer/anaconda/envs/_build/mkspecs
```
where the only relevant item from `build.sh` is that it calls `qmake` which finds `~/anaconda/bin/qmake` first on the path.

Is it likely to mess anything up with Anaconda if ensure the system `qmake` is first on the path? It seems not nice for developers needing to use qmake to be surprised to find an Anaconda-provided qmake is first on the path... would be good if the Anaconda installer did not do this.

**Comment by dondiego87** [comment]
I'm having this same issue. Any ideas?

**Comment by ccordoba12** [comment]
@dondiego87, this is going to be fixed in the next Anaconda release, where a new Qt version is going to be released.

**Comment by ccordoba12** [comment]
This was fixed in Anaconda 2.2 by adding a `qt.conf` file that makes `qmake` to correctly locate `mkspecs`.
